### PR TITLE
[Websocket] Websocket migration followup

### PR DESCRIPTION
**Pull request by Herklos**
- Adding new channels : 
  - Book Ticker
  - Mini Ticker
  - Liquidations
- Websocket features : 
  - keep alive

Depends on https://github.com/Drakkar-Software/OctoBot-Trading/pull/143 and https://github.com/Drakkar-Software/OctoBot-Trading/pull/141

### CODE REVIEW

File: octobot_trading/producers/orders_updater.py [hunk 1]
```diff
@@ -91,7 +81,8 @@ class CloseOrdersUpdater(OrdersProducer):
                         limit=self.ORDERS_UPDATE_LIMIT)
 
                     if close_orders:
-                        await self.push(orders=self._cleanup_close_orders_dict(close_orders),
+                        await self.push(orders=list(map(self.channel.exchange_manager.exchange.clean_order,
```
Comment (GuillaumeDSM) [inline octobot_trading/producers/orders_updater.py:84]: 👍

File: octobot_trading/producers/prices_updater.py [hunk 2]
```diff
@@ -127,7 +127,7 @@ def __init__(self, channel):
                 push(symbol=symbol,
                      funding_rate=funding_rate[ExchangeConstantsFundingColumns.FUNDING_RATE.value],
                      next_funding_time=funding_rate[ExchangeConstantsFundingColumns.NEXT_FUNDING_TIME.value],
-                     timestamp=funding_rate[ExchangeConstantsFundingColumns.TIMESTAMP.value])
+                     timestamp=funding_rate[ExchangeConstantsFundingColumns.LAST_FUNDING_TIME.value])
```
Comment (GuillaumeDSM) [inline octobot_trading/producers/prices_updater.py:130]: 👍

File: octobot_trading/producers/ticker_updater.py [hunk 3]
```diff
@@ -69,6 +71,28 @@ def _cleanup_ticker_dict(self, ticker):
...
+                    ExchangeConstantsMiniTickerColumns.LOW_PRICE.value:
+                        ticker[ExchangeConstantsTickersColumns.LOW.value],
+                    ExchangeConstantsMiniTickerColumns.OPEN_PRICE.value:
+                        ticker[ExchangeConstantsTickersColumns.OPEN.value],
+                    ExchangeConstantsMiniTickerColumns.CLOSE_PRICE.value:
+                        ticker[ExchangeConstantsTickersColumns.CLOSE.value],
+                    ExchangeConstantsMiniTickerColumns.VOLUME.value:
+                        ticker[ExchangeConstantsTickersColumns.BASE_VOLUME.value],
+                    ExchangeConstantsMiniTickerColumns.TIMESTAMP.value:
+                        ticker[ExchangeConstantsTickersColumns.TIMESTAMP.value]})
```
Comment (GuillaumeDSM) [inline octobot_trading/producers/ticker_updater.py:92]: 💯

File: octobot_trading/producers/orders_updater.py [hunk 4]
```diff
@@ -79,7 +69,7 @@ def _cleanup_open_orders_dict(self, open_orders):
 
 class CloseOrdersUpdater(OrdersProducer):
     CHANNEL_NAME = ORDERS_CHANNEL
-    ORDERS_REFRESH_TIME = 2  # TODO = 10
+    ORDERS_REFRESH_TIME = 72
```
[the remark this answers is not in the document]
Comment (Herklos) [reply octobot_trading/producers/orders_updater.py:72]: Exactly so I don't think 2 sec is really needed

**Pull request merged**